lm-logs/templates/configmap.yaml

```diff
@@ -65,7 +66,21 @@ data:
         </pattern>
       </parse>
     </source>
-    
+
```

Comment:
What is the customer does want this multiline start regex, what if it wants use the default `\n`?